app/commands/unlock_command.rb

```diff
@@ -9,8 +9,26 @@ def run
       return Slash.reply(ValidationErrorMessage.build(record: env)) if env.invalid?
 
       slashdeploy.unlock_environment(user.user, env)
-      Slash.say UnlockedMessage.build \
-        environment: env
+      new_active_lock = slashdeploy.give_lock_to_next_user(env)
+
+      if new_active_lock
+        Thread.new do
```

Comment:
Let's not do this out of band for now. Mainly because, if this throws an exception, it'll be swallowed.

It may be good to follow up later with a `slack` implementation that uses something like celluloid to send in the background.